install/ui/src/freeipa/navigation/menu_spec.js

```diff
@@ -151,7 +151,21 @@ var nav = {};
                     ]
                 },
                 { entity: 'otptoken' },
-                { entity: 'radiusproxy' }
+                { entity: 'radiusproxy' },
+                {
+                    entity: 'certmaprule',
```

Comment:
Maybe more a question for UX person: is the position of the menu item on good place? Right now the order is: 
* Certificates
* OTP Tokens
* RADIUS Servers
* Certificate Identity Mapping Rules

Following order would be more natural to me:
* Certificates
* Certificate Identity Mapping Rules
* OTP Tokens
* RADIUS Servers

But I don't insist on it.